pkg/trtl/trtl.go

```diff
@@ -525,6 +615,11 @@ func (h *TrtlService) Cursor(in *pb.CursorRequest, stream pb.Trtl_CursorServer)
 
 		// Count the number of messages successfully sent
 		nMessages++
+
+		// TODO: Prometheus
+		// If in.Prefix is nil, nMessages will be all the objects in in.Namespace so we
+		// could use this opportunity to update our Prometheus counter if we can find
+		// out how to call something like update on the counter rather than increment
```

Comment:
Here's a place where I think we have an opportunity to count objects in the namespace, though prometheus likes to increment, not count. Maybe we can figure out a workaround?

Reply:
Can a prometheus `Gauge` count?